src/components/FilmsHeroImage/FilmsHeroImage.tsx

```diff
@@ -0,0 +1,31 @@
+import * as S from "./FilmsHeroImage.Styled";
+
+interface BackgroundProps {
+	heroImage: {
+		backdrop_path: string;
+		original_title: string;
+		overview: string;
+	};
+	alt: string;
+}
+
+function FilmsHeroContainer({ heroImage, alt }: BackgroundProps) {
+	if (!heroImage) {
```

Comment:
Szkoły są różne, jedni nakazaliby warunkowe renderowanie FilmsHeroContainer w miejscu użycia jeżeli nie ma heroImage, drudzy poprą to rozwiązanie. Warto wiedzieć, czy warto zmieniać - w sumie nie wiem haha